homeassistant/components/alarm_control_panel/icons.json

```diff
@@ -3,7 +3,7 @@
     "_": {
       "default": "mdi:shield",
       "state": {
-        "armed_away": "mdi:shield-lock",
+        "armed_away": "mdi:exit-run",
```

Comment:
I'm not sure I think this is better. I'll leave it to a second opinion.

Reply:
Based on a quick survey of other panels, Xfinity and Ring use the running convention. Simplisafe, Vivint and Frontpoint uses the word "away". ADT uses the shield, with a check. Abode uses a lock icon.

Reply:
@radiocolin Could you clarify what "survey of other panels" means? Do you mean you found some images of commercial alarm systems and classified how they signal the "armed away" state?

Reply:
For context, this is the current icon, i.e. without the changes in this PR:
![image](https://github.com/user-attachments/assets/e1dc8ccf-5ab2-473c-bf70-ea497737a491)

This is the icon this PR suggests we change to:
![image](https://github.com/user-attachments/assets/a84c6fa9-f276-4741-a5f1-1e2abc07e783)

I also don't think the proposal is an improvement.

Reply:
> @radiocolin Could you clarify what "survey of other panels" means? Do you mean you found some images of commercial alarm systems and classified how they signal the "armed away" state?

Correct. Seems like there's nothing particularly standard. 

My thought process here is that we know the icon is for "armed", but that a shield and lock doesn't do anything to convey the "away" portion. A house is used for home (conveys the home portion) and an unlock is used for disarmed.

Reply:
I fully agree the existing icon is not great.
The proposed icon is very similar to the pictogram used for emergency exits though, so I don't think it's an improvement.

Reply:
The ideal icon would probably be a shield + exiting person combo icon, which I don't think exists in the current set.

Reply:
I personally think that there is no real nice alternative here. Yes, the current icon does not really match with "away", but it does introduce the consistency where every `armed` state, has a shield. So it's easy to link shield == armed. And since `exit-run` does not really correlate with `armed`, I think there is no ideal choice that works on both sides.

And for me the consistency with shield == armed weighs more than exposing the "away" better

Reply:
> The ideal icon would probably be a shield + exiting person combo icon, which I don't think exists in the current set.

Yeah. This is the icon used by the alarm system in my house:
![image](https://github.com/user-attachments/assets/d6fb03a0-e87a-48b2-ac39-7344eb197a66)

The company's newer products seem to use these icons though, where the half-filled shield means armed home and the checked shield means armed away:
![image](https://github.com/user-attachments/assets/cc9a268d-156a-4364-9cfb-443bec20f807)